### PR TITLE
feat(env-var): add env support for cloud urls

### PR DESCRIPTION
**Pull request by hoop33**
**Before submitting a pull request,** please make sure the following is done:

1. Fork [the repository](https://github.com/availity/availity-react) and create your branch from `master`.
2. Run `yarn` in the repository root.
3. If you've fixed a bug or added code that should be tested, add tests!
4. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch TestName` is helpful in development.
5. Make sure your code passed the conventional commits check. Read more about [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)

### CODE REVIEW

File: packages/env-var/src/index.js [hunk 1]
```diff
@@ -1,28 +1,91 @@
...
+
 export function setEnvironments(envs, override) {
   if (override) {
     environments = envs;
   } else {
     Object.assign(environments, envs);
   }
 }
+
+export function setSpecificEnvironments(envs, override) {
```
Comment (nylon22) [inline packages/env-var/src/index.js:61]: Can we get an accompanying documentation change for this function? https://github.com/Availity/sdk-js/blob/master/docs/source/features/env-var.md

Comment (hoop33) [reply packages/env-var/src/index.js:61]: Added documentation

File: docs/source/features/env-var.md [hunk 2]
```diff
@@ -99,3 +99,74 @@ setEnvironments({
     myEnv: ['custom-stuff-here'],
 });
 ```
+
+### getSpecificEnv
+
+Get the specific current environment, without rolling up to the general environment. Whereas `envVar` treats the `t01` environment as `test`, for example, `getSpecificEnv` returns `'t01'` for the `t01` environment.
+
+```js
+import { getCurrentEnv } from '@availity/env-var';
```
Comment (nylon22) [inline docs/source/features/env-var.md:108]: s/b

```
import { getSpecificEnv } from '@availity/env-var';
```

File: docs/source/features/env-var.md [hunk 3]
```diff
@@ -99,3 +99,74 @@ setEnvironments({
...
+#### Example
+
+```js
+import { getCurrentEnv } from '@availity/env-var';
+
+/*
+depending on the environment this code runs in, specificEnv would be something different,
+like 't01' or 'stg' or 'prod'
+*/
+const specificEnv = getSpecificEnv();
```
Comment (nylon22) [inline docs/source/features/env-var.md:132]: s/b

```
import { getSpecificEnv } from '@availity/env-var';
```

File: docs/source/features/env-var.md (hunk 2 above)
Comment (hoop33) [reply docs/source/features/env-var.md]: fixed

File: docs/source/features/env-var.md (hunk 3 above)
Comment (hoop33) [reply docs/source/features/env-var.md:132]: fixed

**Pull request merged**